### ISSUE TITLE
I've never had a problem before with using Advanced X-Ray in 1.12.2 and now I suddenly do.

### ISSUE THREAD

**Issue by APiercingEmbarrassment** [AdvancedXRay/XRay-Mod#189]
**Describe the bug**
I can't even get past the 1.12.2 Forge loading screen before my game crashes and gives me this: The game crashed whilst there was a severe problem during mod loading that has caused the game to fail
Error: net.minecraftforge.fml.common.LoaderExceptionModCrash: Caught exception from XRay (xray)
Exit Code: -1

**To Reproduce**
Pictures of the mods I use
![mods 11_15_2021 6_16_40 PM](ht
![mods 11_15_2021 6_17_22 PM](https://user-images.githubusercontent.com/94414642/141872855-a2d7af9b-8f8b-486f-bdd9-31bd805a00b8.png)
![mods 11_15_2021 6_16_49 PM](https://user-images.githubusercontent.com/94414642/141872865-202d25f9-3ac7-4546-b068-779e211e2c32.png)
![mods 11_15_2021 6_17_14 PM](https://user-images.githubusercontent.com/94414642/141872869-48d67dce-9979-4ef4-8c9c-3e0ed0aaf96e.png)
tps://user-images.githubusercontent.com/94414642/141872841-5433d87a-5397-4781-9aff-00050f172b40.png)

What I expected to happen was it to run perfectly fine, but since I came back from 1.16.5 to take a break from modded 1.16.5 my 1.12.2 version of the mod has been acting up and I've never had a problem with it and these mods before, and now I suddenly do?

Here's the code as well: https://gist.github.com/APiercingEmbarrassment/84ad022c7333828de6ec6d844dac94d5

**Comment by APiercingEmbarrassment** [comment]
Here's one more picture as well
![mods 11_15_2021 6_16_40 PM](https://user-images.githubusercontent.com/94414642/141873347-b4a670cc-f755-40b6-915d-654872d58290.png)

**Comment by MichaelHillcox** [comment]
Delete the config file in /config/xray/some.json file I can't remember the name of the json file. Should fix your issue

**Comment by APiercingEmbarrassment** [comment]
> Delete the config file in /config/xray/some.json file I can't remember the name of the json file. Should fix your issue

The only json file was block_store, I deleted it and it loaded right up. Although I was using the 1.16.5 version of the mod in a 1.16.5 modpack. Could it be that the blocks I added for it to find in 1.16.5 could have caused this since those blocks would be stored in the file menu of blocks that it was set to find only in 1.16.5 mods, but since I didn't delete those blocks out of x-ray before going back down to 1.12.2, it caused the game to crash because those mods were no longer in the mods folder and so those blocks didn't exist?
